modules/schematics/src/action/files/__name@dasherize@if-flat__/__name@dasherize__.actions.spec.ts.template

```diff
@@ -1,7 +1,7 @@
-import * as <%= classify(name) %>Actions from './<%= dasherize(name) %>.actions';
+import * as from<%= classify(name) %> from './<%= dasherize(name) %>.actions';
```

Comment:
It seems that this import should remain the same because actions are imported, not reducer 👀 

```suggestion
import * as <%= classify(name) %>Actions from './<%= dasherize(name) %>.actions';
```

Reply:
That's a good point, I just re-used the original without giving it a thought.
But does it even make sense to write a spec for actions with action creators?

Reply:
Agree 👍  Testing [creators with props factory](https://github.com/ngrx/platform/blob/master/modules/store/src/action_creator.ts#L22) makes sense. Otherwise, it's unnecessary IMO.

Reply:
Removed in [6de2ef9](https://github.com/ngrx/platform/pull/3311/commits/6de2ef9aacf84f22d7f43167941d69190a95ff63)